assets/src/components/activities/common/hints/delivery/HintsDeliveryConnected.tsx

```diff
@@ -28,7 +28,7 @@ const shouldShow = (
 ) => {
   if (graded) return false;
   if (surveyId !== null) return false;
-  if (!correct) return true;
+  if (!correct || correct) return true;
```

Comment:
This always evaluates to true because `(!true || true) == (false || true) == true` so we can just replace it with `return true` and remove the other return below this one, right?

Reply:
You're right! Thanks!